modules/nodes-cluster-overcommit-project-disable.adoc

```diff
@@ -7,29 +7,60 @@
 [id="nodes-cluster-overcommit-project-disable_{context}"]
 = Disabling overcommitment for a project
 
+ifndef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 When enabled, overcommitment can be disabled per-project. For example, you can allow infrastructure components to be configured independently of overcommitment.
+endif::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
 
-.Procedure
+ifdef::openshift-rosa,openshift-rosa-hcp,openshift-dedicated[]
+If required by your use case, you can disable overcommitment on any project that is not managed by Red Hat. For a list of projects that cannot be modified, see "Red Hat Managed resources" in _Support_.
```

Comment:
I am not sure where this style comes from, but people seem to be using the non-breaking space thingy within Red Hat.
```suggestion
If required by your use case, you can disable overcommitment on any project that is not managed by Red{nbsp}Hat. For a list of projects that cannot be modified, see "Red Hat Managed resources" in _Support_.
```